src/selectors/tags.js

```diff
@@ -1,10 +1,29 @@
+import { getCategoryById } from './categories'
+
+export const getTagById = (state, tagId) => {
+  const matching = state.tags.find(tag => tag.id === tagId)
+  if (matching === undefined) return null
+  return Object.assign(
+    {},
+    matching,
+    {
+      category: getCategoryById(matching.categoryId),
+      categoryId: undefined
```

Comment:
@amandahogan The idea here was to replace the `categoryId` property with a copy of the category object itself. Setting `categoryId` to `undefined` deletes that property as it is no longer required since the category itself has been added.

My thinking was that Tag components would need access to the category object in order to know what colour they are. Turns out when I came to implement displaying the tags (in another pull request) it isn't necessary for the tag objects to contain that info because they are already getting passed the category as a separate prop.